libnd4j/CMakeLists.txt

```diff
@@ -50,25 +50,6 @@ if(NOT CUDA_BLAS)
     # we need this definition to avoid global memory use within mkldnn
     add_definitions(-DMKLDNN_ENABLE_CONCURRENT_EXEC=true)
 
-    # old mkl-dnn entry
-    if (NOT "${MKLDNN_PATH}" STREQUAL "")
-        message("We have manually provided MKLDNN: ${MKLDNN_PATH}")
-        set(HAVE_MKLDNN 1)
-        include_directories(${MKLDNN_PATH}/include/)
-        link_directories(${MKLDNN_PATH} ${MKLDNN_PATH}/lib/)
-        IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-            set(MKLDNN_LIBRARIES mkldnn mklml_intel)
-        else()
-            set(MKLDNN_LIBRARIES mkldnn mklml)
-        endif()
-    elseif (NOT "${OPENBLAS_PATH}" STREQUAL "")
```

Comment:
Don't remove OpenBLAS

Reply:
It's not removed, if OPENBLAS_PATH is provded - it'll be used

Reply:
Why did you remove those lines though? I'm pretty sure OpenBLAS isn't going to get linked

Reply:
Mmm, let me double check then. The plan was to keep OPENBLAS_PATH if it was provided

Reply:
Yep, you're right. Missed else part. Restored it back.